eth-contracts/scripts/geth/deployAll.js

```diff
@@ -0,0 +1,131 @@
+// We require the Buidler Runtime Environment explicitly here. This is optional
+// but useful for running the script in a standalone fashion through `node <script>`.
+// When running the script with `buidler run <script>` you'll find the Buidler
+// Runtime Environment's members available in the global scope.
+
+function sleep(ms) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function main() {
+  // Buidler always runs the compile task when running scripts through it.
+  // If this runs in a standalone fashion you may want to call compile manually
+  // to make sure everything is compiled
+  // await bre.run('compile');
+
+  //ERC20
+  const ERC20 = await ethers.getContractFactory(
+    "contracts/test/ERC20.sol:ERC20"
+  );
+  const ERC20Deploy = await ERC20.deploy();
+  await ERC20Deploy.deployed();
+  const ERC20DeployAddr = ERC20Deploy.address;
+  console.log("ERC20 deployed to:", ERC20DeployAddr);
+  console.log("waiting for block confirmations, about 1 minute");
+  await sleep(60 * 1000);
+
+  const contractBalance = await ERC20Deploy.callStatic.balanceOf(
+    "0x17c4b5CE0605F63732bfd175feCe7aC6b4620FD2"
```

Comment:
what is this address stands for?

Reply:
> what is this address stands for?

the erc20 contract deployer, just for test.  It been removed